cosmo_tester/test_suites/image_based_tests/cfy_manager_test.py

```diff
@@ -63,7 +64,14 @@ def test_cfy_manager_configure(image_based_manager, logger, tmpdir,
     image_based_manager.put_remote_file_content(REMOTE_HOOKS_PATH, NEW_HOOKS)
 
     logger.info('Running `cfy_manager configure`...')
-    image_based_manager.run_command('cfy_manager configure')
+    image_based_manager.run_command(
```

Comment:
Why do we do this? Am I missing the place where we're changing the password in this test?

Reply:
Well, the admin_password is blank in the config.yaml of the manager. Even though there is this section at the end:
```yaml
security:
  admin_username: admin
  admin_password: admin
```

The problem is that it's not under the `manager` section, and I couldn't find out where it is generated (maybe as part of the image generation?).

Reply:
Base install config is generated here: https://github.com/cloudify-cosmo/cloudify-system-tests/blob/master/cosmo_tester/framework/test_hosts.py#L345

That puts it in the right place - I wonder if some of the config rewriting silliness is mangling it?

Reply:
We fixed this in the builder, marking resolved.